src/Config.php

```diff
@@ -322,6 +322,12 @@ protected function parseGeneral()
         // Read the config and merge it. (note: We use temp variables to prevent
         // "Only variables should be passed by reference")
         $tempconfig = $this->parseConfigYaml('config.yml');
+        if(isset($_SERVER['ENVIRONMENT']))
```

Comment:
The other problem I have with this approach is that it is putting config back into the request cycle to finish … @CarsonF any ideas on approach here?